d2g_test.go

```diff
@@ -59,6 +59,9 @@ func TestDecisionTask(t *testing.T) {
 	      "chainOfTrust": true,
 	      "taskclusterProxy": true
 	    },
+		"capabilities": {
+			"privileged": true
+		},
```

Comment:
Indentation!

Reply:
Omg lol sorry!

Reply:
Fixed in https://github.com/taskcluster/d2g/pull/3/commits/202aa4a8c52f23afd88c7f538beee174a29bcfff :)